wp-content/civi-extensions/goonjcustom/Civi/UrbanPlannedVisitService.php

```
@@ -679,6 +680,68 @@ public static function assignCenterGroupToIndividual(string $op, string $objectN
 
   }
 
+  /**
+   * Assigns a contact/institute to the state/province group of a given center.
+   *
+   * @param int $contactId ID of the individual contact.
+   * @param int $centerProvinceId Contact ID of the center to get state/province.
+   */
+  public static function assignBackendCenterGroupToIndividual(string $op, string $objectName, $objectId, &$objectRef) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$objectId'. (undefined)

(UnusedFormalParameter)